ciphey/basemods/Decoders/dna.py

```diff
@@ -0,0 +1,50 @@
+from typing import Optional, Dict, List
+
+from ciphey.iface import Config, ParamSpec, T, U, Decoder, registry, Translation
+
+from loguru import logger
+
+import re
+
+
+@registry.register
+class Dna(Decoder[str, str]):
+    def decode(self, ctext: T) -> Optional[U]:
+        """
+        Performs DNA decoding
+        """
+        logger.trace("Attempting DNA decoder")
+        ctext_decoded = ""
+        ctext = re.sub(r"[,;:\-\s]", "", ctext)
+        ctext = " ".join(ctext[i : i + 3] for i in range(0, len(ctext), 3))
+        ctext_split = ctext.split(" ")
+
+        try:
```

Comment:
I don't like this try except :( 

You can check if a string exists in a dictionary with:

```python
x = {"hello": 6}
keys = x.keys()
if "food" in keys:
```

Other than that, it's good! :)

Reply:
Okay, I removed the try except. I tested it, it works good.